content/src/content/jcr_root/apps/core/wcm/components/page/v2/page/page.html

```diff
@@ -19,6 +19,11 @@
       data-sly-use.footer="footer.html"
       data-sly-use.redirect="redirect.html">
     <head data-sly-call="${head.head @ page = page}"></head>
+
+    <script type="text/javascript">
```

Comment:
Eventually this script will need to be conditionally added based on whether the data layer is enabled, or not. Could be something like:
```
<script data-sly-test=${page.dataLayerJson} type="text/javascript">
   // my script
</script>
```